packages/esm-patient-search-app/src/patient-search-result/result-card/result-card.component.tsx

```diff
@@ -0,0 +1,127 @@
+import { age, ExtensionSlot, formatDate, navigate, parseDate, useVisit } from '@openmrs/esm-framework';
+import { Button, ClickableTile } from 'carbon-components-react';
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from './result-card.scss';
+import ChevronDown16 from '@carbon/icons-react/es/chevron--down/16';
+import ChevronUp16 from '@carbon/icons-react/es/chevron--up/16';
+import OverflowMenuVertical16 from '@carbon/icons-react/es/overflow-menu--vertical/16';
+import CustomOverflowMenuComponent from './overflow-menu.component';
+import ContactDetails from '../contact-details/contact-details.component';
+import { capitalize } from 'lodash-es';
+
+interface ResultCardProp {
+  patient: fhir.Patient;
+  onSearchResultClick: (patientUuid: string) => void;
+  closeSearchResultsPanel: () => {};
+}
+
+const ResultCard: React.FC<ResultCardProp> = ({ patient, onSearchResultClick, closeSearchResultsPanel }) => {
+  const { t } = useTranslation();
+  const overFlowMenuRef = React.useRef(null);
```

Comment:
Let's just always call it the "actions menu" and get rid of the "overflow menu" name altogether. Also see https://github.com/openmrs/openmrs-esm-patient-chart/pull/667/files